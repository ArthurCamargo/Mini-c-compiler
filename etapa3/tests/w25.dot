digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n1 -> n5;
n2;
n2 [label="a"];
n3;
n3 -> n6;
n3 -> n7;
n6;
n6 [label="a"];
n7;
n7 [label="b"];
n3 [label="="];
n4;
n4 -> n8;
n4 -> n9;
n8;
n8 [label="a"];
n9;
n9 [label="3"];
n4 [label="="];
n5;
n5 -> n10;
n5 -> n11;
n5 -> n12;
n10;
n10 [label="c"];
n11;
n11 [label="4"];
n12;
n12 -> n13;
n12 -> n14;
n13;
n13 [label="c"];
n14;
n14 [label="5"];
n12 [label="="];
n5 [label="="];
n1 [label="if"];
n0 [label="f"];

}