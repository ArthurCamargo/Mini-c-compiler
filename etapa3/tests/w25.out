0x560848381350
0x560848381350, 0x5608483817c0
0x5608483817c0
0x5608483817c0, 0x560848381530
0x5608483817c0, 0x560848381660
0x5608483817c0, 0x560848381770
0x5608483817c0, 0x5608483818d0
0x560848381530
0x560848381530 [label="a"];
0x560848381660
0x560848381660, 0x5608483815a0
0x560848381660, 0x560848381610
0x5608483815a0
0x5608483815a0 [label="a"];
0x560848381610
0x560848381610 [label="b"];
0x560848381660 [label="="];
0x560848381770
0x560848381770, 0x5608483816d0
0x560848381770, 0x560848381720
0x5608483816d0
0x5608483816d0 [label="a"];
0x560848381720
0x560848381720 [label="3"];
0x560848381770 [label="="];
0x5608483818d0
0x5608483818d0, 0x560848381830
0x5608483818d0, 0x560848381880
0x5608483818d0, 0x5608483819e0
0x560848381830
0x560848381830 [label="c"];
0x560848381880
0x560848381880 [label="4"];
0x5608483819e0
0x5608483819e0, 0x560848381940
0x5608483819e0, 0x560848381990
0x560848381940
0x560848381940 [label="c"];
0x560848381990
0x560848381990 [label="5"];
0x5608483819e0 [label="="];
0x5608483818d0 [label="="];
0x5608483817c0 [label="if"];
0x560848381350 [label="f"];
